app/components/channel_item/index.ts

```diff
@@ -40,7 +41,7 @@ const enhance = withObservables(['channel', 'showTeamName'], ({channel, database
             if (!mc) {
                 return of$(false);
             }
-            return observeIsMutedSetting(mc);
+            return observeIsMutedSetting(mc) || of$(isCategoryMuted);
```

Comment:
The left side always returns truthy, you would need to pipe it. Also does the category takes presefence over the channel, no?